internals/daemon/daemon_test.go

```diff
@@ -60,6 +61,11 @@ type daemonSuite struct {
 var _ = Suite(&daemonSuite{})
 
 func (s *daemonSuite) SetUpTest(c *C) {
+	err := reaper.Start()
```

Comment:
Could this change cause the previous issue again when we call shutdown during tests (on-reaper aware)?

https://github.com/canonical/pebble/pull/279

With reaper top level, I believe we could simply replace the ```cmd.CombinedOutput``` in the daemon.go (```systemdModeReboot```) with the reaper alternative for now?

Reply:
Done (also in `systemd.go`).